lib/Elevate/Components/SSH.pm

```diff
@@ -0,0 +1,51 @@
+package Elevate::Components::SSH;
+
+=encoding utf-8
+
+=head1 NAME
+
+Elevate::Components::SSH
+
+Ensure that the sshd config file has 'PermitRootLogin' set to 'yes'
+if it is not set.
+
+=cut
+
+use cPstrict;
+
+use Elevate::Constants ();
+
+use Cwd           ();
+use File::Slurper ();
+use Log::Log4perl qw(:easy);
+
+use parent qw{Elevate::Components::Base};
+
+sub pre_leapp ($self) {
+
+    my $sshd_config = q[/etc/ssh/sshd_config];
+
+    my $setup = File::Slurper::read_binary($sshd_config) // '';
```

Comment:
I'm twitching that you're not doing read_text because the encoding of the SV will be byte not utf8. Just the same, I'd probably make the same decision.

Reply:
That is what we originally did (and do) in the SSH blocker.

Reply:
yep. It's right. I'm still twitching :) Encoding sucks.